source/agora/consensus/protocol/Nominator.d

```diff
@@ -142,6 +142,9 @@ public extern (C++) class Nominator : SCPDriver
     /// Delegate called when a block is to be externalized
     public extern (D) string delegate (in Block) @safe acceptBlock;
 
+    /// Delegate called when a block header is to be updated with signatures
+    public extern (D) void delegate (const(BlockHeader)) @safe acceptHeader;
+
```

Comment:
The proliferation of those delegates starts to be a concern.

Reply:
Would be quite a big refactor to avoid this one.